Makefile

```
@@ -0,0 +1,2 @@
+PATH := ~/.solc-select/artifacts/solc-0.8.16:~/.solc-select/artifacts/solc-0.5.12:~/.solc-select/artifacts:$(PATH)
```

You don't need to specify every compiler version if you use `wildcard`:
```suggestion
PATH := $(wildcard ~/.solc-select/artifacts/*):$(PATH)
```